docs/zh-CN/docs/data-operate/import/import-scenes/external-storage-load.md

```diff
@@ -124,7 +124,7 @@ Hdfs load 创建导入语句，导入方式和[Broker Load](../../../data-operat
 * 数据量在 几十到百GB 级别。
 
 ### 准备工作
-1. 准本AK 和 SK
+1. 准备 AK 和 SK
```

Comment:
please fix the english doc also. In english doc its mentioned as "Standard AK and SK First".
change this also to "Prepare AK and SK First"

Reply:
done